### ISSUE TITLE
i-bem.js как standalone

### ISSUE THREAD

**Issue by JiLiZART** [bem/bem-forum-content-ru#846]
А не задумывались ли вы о самостоятельной i-bem.js?
Так сказать лайт версия стека БЭМ. Очень часто встречаю файл scripts.js, main.js, с наиполнейшей лапшой из jquery плагинов, без ленивой инициализации и всех крутых штук которые умеет i-bem.js

**Comment by zxqfox** [comment]
Ты что, на святой посягаешь? Еще про скажи, что jquery тебе вообще не нужен!

**Comment by JiLiZART** [comment]
@zxqfox идея в том чтобы предоставить хорошую модульность, в купе с методологией именования классов. Не всем по душе полный стек, не все умеют настраивать enb. А так было бы довольно эффективно, скажем в среднем проекте использовать i-bem.js + jquery с плагинами и получать хорошую модульную архитектуру из коробки. Плюс так же собирать это можно чем угодно.

**Comment by tadatuta** [comment]
@JiLiZART 

Вот же: https://github.com/bem/bem-core-dist (если ставить через `bower` или `git`).
И развернута на нашем CDN: https://yastatic.net/bem-core/latest/desktop/bem-core.js (подробнее про схему урлов на CDN см. https://ru.bem.info/libs/bem-components/v2.4.0/#Подключение-файлов-с-cdn

@zxqfox Алексей, прекратите троллировать! ;)

**Comment by zxqfox** [comment]
У нас есть в разработке bem-core v3 (это, по сути, и есть i-bem, разбитый на модули).

Или ты хочешь еще сильнее разбить блок i-bem? Есть какие-то конкретные предложения?

**Comment by JiLiZART** [comment]
@tadatuta плохой у вас маркетинг :) Только через пост на форуме узнал.
Даже здесь https://ru.bem.info/libs/bem-core/v2.8.0/ ничего про это не написано. Очень нехватает в таких местах quick start и всех поддерживаемых способов установки

**Comment by tadatuta** [comment]
Да, я давно всем обещаю подраздел на bem.info про dist-сборку каждой библиотеки, но все никак не сделаю :(

Подробно есть описание только для `bem-components` (по ссылке из моего предыдущего комментария). В качестве хилой отмазки могу лишь сказать, что был пост на форуме при первом релизе: https://ru.bem.info/forum/469 и напоминание в рамках дайджеста https://ru.bem.info/forum/787

**Comment by zxqfox** [comment]
Маркетинг :pray: :cloud:
